### PR TITLE
ServiceRequest.priority changed to MustSupport with obligations (FHIR-46939)

### PR DESCRIPTION
**Pull request by JayMurdoch**
Fix for [FHIR-46939](https://jira.hl7.org/browse/FHIR-46939)

* Add MS and associated obligations to AU eRequesting ServiceRequest.priority. 
* Change log entry

### CODE REVIEW

File: input/pagecontent/changes.md [hunk 1]
```diff
@@ -9,3 +9,5 @@ This change log documents the significant updates and resolutions implemented fr
 - New profiles: 
   - AU eRequesting Task ([FHIR-46947](https://jira.hl7.org/browse/FHIR-46947), [FHIR-46950](https://jira.hl7.org/browse/FHIR-46950), [FHIR-46956](https://jira.hl7.org/browse/FHIR-46956), [FHIR-46958](https://jira.hl7.org/browse/FHIR-46958), [FHIR-46959](https://jira.hl7.org/browse/FHIR-46959), [FHIR-47081](https://jira.hl7.org/browse/FHIR-47081), [FHIR-47082](https://jira.hl7.org/browse/FHIR-47082), [FHIR-47083](https://jira.hl7.org/browse/FHIR-47083), [FHIR-47084](https://jira.hl7.org/browse/FHIR-47084), [FHIR-47086](https://jira.hl7.org/browse/FHIR-47086), [FHIR-47102](https://jira.hl7.org/browse/FHIR-47102).
   - AU eRequesting Coverage ([FHIR-46848](https://jira.hl7.org/browse/FHIR-46848))083](https://jira.hl7.org/browse/FHIR-47083), [FHIR-47084](https://jira.hl7.org/browse/FHIR-47084), [FHIR-47086](https://jira.hl7.org/browse/FHIR-47086), [FHIR-47102](https://jira.hl7.org/browse/FHIR-47102).
+- Made the following changes to AU eRequesting ServiceRequest
+  - added Must Support and cardinality 0..1 to ServiceRequest.priority [FHIR-46939](https://jira.hl7.org/browse/FHIR-46939)
```
Comment (dbojicic) [inline input/pagecontent/changes.md:13]: Please remove 'and added cardinality' to make it "added Must Support to  ServiceRequest.priority [FHIR-46939](https://jira.hl7.org/browse/FHIR-46939)". We did not change the cardinality.

**Review by dbojicic — CHANGES_REQUESTED** [review]
Thanks @JayMurdoch, a minor change requested.

File: input/fsh/au-erequesting-servicerequest.fsh [hunk 2]
```diff
@@ -188,6 +188,19 @@ Description: "This profile sets minimum expectations for a ServiceRequest resour
 * reasonCode ^extension[http://hl7.org/fhir/StructureDefinition/obligation][4].extension[code].valueCode = #SHALL:no-error
 * reasonCode ^extension[http://hl7.org/fhir/StructureDefinition/obligation][4].extension[actor][0].valueCanonical = "http://hl7.org.au/fhir/ereq/ActorDefinition/au-erequesting-actor-patientaccess"
 
+* priority MS
+* priority from http://hl7.org/fhir/ValueSet/request-priority (required)
```
Comment (dbojicic) [inline input/fsh/au-erequesting-servicerequest.fsh:192]: This is already a required binding in the core FHIR ServiceRequest profile, no need to replicate it here. Please remove.

**Review by dbojicic — CHANGES_REQUESTED** [review]
Thanks @JayMurdoch, just one more change. Sorry I missed it last time.

File: input/fsh/au-erequesting-servicerequest.fsh (hunk 2 above)
Comment (JayMurdoch) [reply input/fsh/au-erequesting-servicerequest.fsh]: Removed, thanks.

**Review by dbojicic — APPROVED** [review]

**Review by michael-wilson-au — APPROVED** [review]

**Pull request merged**